ci/scripts/audit.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -eux
+
+pushd babbage
+  cd src/main/web
```

Comment:
Why not merge the cd into the pushd?
```suggestion
pushd babbage/src/main/web
```

Reply:
Good shout; I don't see any reason why not to